templates/vmware-images.html

```diff
@@ -0,0 +1,155 @@
+              <label for="Consent_to_Processing__c"></label>
+            </div>
+            <p>In submitting this form, I confirm that I have read and agree to <a href="https://www.ubuntu.com/legal/data-privacy">Canonical&rsquo;s Privacy Notice</a> and <a href="https://www.ubuntu.com/legal/data-privacy/esxi">Privacy Policy</a>.
+            <div class="p-form__group mktField">
+              <button type="button" class="mktoButton p-button--neutral close-modal">Cancel</button>
+              <button type="submit" class="mktoButton p-button--positive">Download template</button>
+            </div>
+            <input type="hidden" name="formid" class="mktoField" value="3392">
+            <input type="hidden" name="formVid" class="mktoField" value="3392">
+            <input type="hidden" name="munchkinId" class="mktoField" value="066-EOV-335">
+            <input type="hidden" name="download_asset_url" class="mktoField" value="https://assets.ubuntu.com/v1/f8e9fa9d-packer-maas-1.0.2.tar.xz">
+            <input type="hidden" name="returnURL" value="">
+            <input type="hidden" name="retURL" value="">
+          </form>
+        </div>
+      </div>
+    </div>
+  </div>
+  <div class="row modal-success u-hide">
+    <div class="col-12">
+      <h2>Thank you for registering</h2>
+      <p>
+        <strong>Your download should start automatically</strong>.<br>
+        Problems? <a href="https://assets.ubuntu.com/v1/f8e9fa9d-packer-maas-1.0.2.tar.xz" class="no-icon">Download 'ESXi.iso.packer.template'</a>.
+      </p>
+    </div>
+  </div>
+<style>
+  .no-icon:after {
+    display: none;
+  }
+</style>
+<script src="https://assets.ubuntu.com/v1/4176b39e-serialize.js"></script>
+<script src="https://assets.ubuntu.com/v1/ec520d10-XMLHttpRequest.min.js"></script>
+<script src="https://assets.ubuntu.com/v1/6b7597df-event-listener-polyfill.js"></script>
+<script>
+  var backgroundSubmitHandlerClosure = function(event) {
+    event.preventDefault();
+    var marketoForm = document.getElementById(event.target.id);
+    marketoForm.action = "https://app-sjg.marketo.com/index.php/leadCapture/save2";
+    backgroundSubmit(marketoForm);
+  }
+
+  var backgroundSubmit = function(marketoForm) {
+    var request = new XMLHttpRequest();
+    var submitUrl = marketoForm.getAttribute('action');
+    var formData = serialize(marketoForm);
+    request.open("POST", submitUrl, true);
+
+    //Send the proper header information along with the request
+    request.setRequestHeader("Content-type", "application/x-www-form-urlencoded");
+
+    // Send off the POST request
+    request.send(formData);
+
+    request.addEventListener("readystatechange", function() {
+      if (this.readyState === 4) {
+        showSuccessMessage();
+      }
+    });
+
+    // get the download asset if it exists
+    var download_asset_url = marketoForm.querySelector('input[name=download_asset_url]');
+    if (download_asset_url != null) {
```

Comment:
This should probably be `!==` if you care about checking for null, or just `if (download_asset_url) {`.